src/Joblify.Core.Data/Models/Offer.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Joblify.Core.Data.Models
+{
+    public class Offer
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [Required]
+        public string Title { get; set; }
+
+        public string Description { get; set; }
+
+        public float Price { get; set; }
+        
+        [Required]
+        [ForeignKey("Category")]
+        public int CategoryId { get; set; }
+
+        public Category Category { get; set; }
+
+        [ForeignKey("Trade")]
+        public int TradeId { get; set; }
+
+        public Trade Trade { get; set; }
+
+        public string AvailableTime { get; set; }
```

Comment:
Shouldnt AvailableTime be DateTime? 😄

Reply:
We should clarify what exactly that field is. Is it a time of day like 'in the afternoons' or is it a range of dates? If it's a range of dates then we should have two different fields (?)

Reply:
String is better for us :)